odl/test/trafos/fourier_test.py

```diff
@@ -631,6 +631,53 @@ def char_rect(x):
             assert all_almost_equal(ft.adjoint(ft(char_rect)), discr_rect)
 
 
+def _test_adjoint(operator):
+    """Test adjoint of operator satisfies the definition."""
+    x = noise_element(operator.domain)
+    y = noise_element(operator.range)
```

Comment:
`n / (n // 2 + 1)` sounds very feasible, I saw a change when changing the level of discretization points.